src/Github/GithubInstance.ts

```diff
@@ -42,7 +42,7 @@ export class GithubInstance {
     public static baseOctokitConfig(baseUrl: URL) {
         return {
             userAgent: UserAgent,
-            baseUrl: baseUrl && new URL("/api/v3", baseUrl).toString(),
+            baseUrl: baseUrl.toString().substring(0, baseUrl.toString().length-1),
```

Comment:
What is this doing? Removing the last character without checking what it is?

Reply:
Have added comments to explain how we do things, and removed the horrible URL mangling which wasn't required.